docs/source/installation/macos.rst

```diff
@@ -0,0 +1,100 @@
+MacOS
+=====
+
+For the sake of simplicity, the following instructions assume that you have
+the popular `package manager Homebrew <https://brew.sh>`__ installed. While
+you can certainly also install all dependencies without it, using Homebrew
+makes the process very easy.
```

Comment:
```suggestion
makes the process much easier.
```

Something that's "very easy" to someone familiar with Manim might still be quite difficult to someone unfamiliar, so it's better to compare the difficulty to the alternative manual installation method.